bijection-avro/src/main/scala/com/twitter/bijection/avro/AvroCodecs.scala

```diff
@@ -103,30 +103,53 @@ object SpecificAvroCodecs {
 
   /**
     * Returns Injection capable of serializing and deserializing a compiled avro record using org.apache.avro.io.BinaryEncoder.
+    * @tparam T compiled Avro record
+    * @param schema The record's schema
+    * @return Injection
+    */
+  def toBinary[T <: SpecificRecordBase: ClassTag](schema: Schema): Injection[T, Array[Byte]] = {
```

Comment:
does this actually use the `ClassTag`? I don't see where. If not, since we already dropped it in a previous version, why not correct that issue?